README.md

```
@@ -12,100 +12,65 @@
-
-```mermaid
-flowchart TD
-    Client[클라이언트] -->|Access Token| Gateway
+Edge Authentication 패턴을 적용했습니다.
 
-    subgraph Gateway[API Gateway]
-        G1[토큰 검증 - JWKS 캐싱]
-        G2[Internal Token 발급]
-        G3[라우팅]
-    end
+**구성 요소**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)